Zelda/Zelda.cpp

```diff
@@ -1,1205 +0,0 @@
-
-			while(!InputStream.eof())
-			{
-
-				InputStream.getline(buffer,200);
-				cout << buffer << endl;
-			}
-		}
-	}
-	else
-		cout << "Could not open file";
-	InputStream.close();
-}
-
-
-
-void Game::gameCheck()
-{
-	if(playerPtr->getPrincess_Pointer())
-	{
-		ifstream InputStream("EndWin.txt");
-		if (InputStream.is_open())
-		{
-			HelperFunctions::color (AQUA);
-
-			char buffer[200];
-
-			while(!InputStream.eof())
-			{
-				InputStream.getline(buffer,200);
-				cout << buffer << endl;
-			}
-		}
-		else
-			cout << "Could not open file";
-		InputStream.close();
-	}
-
-
-
-	else
-	{
-
-		ifstream InputStream("EndLose.txt");
-		if (InputStream.is_open())
-		{
-			char buffer[200];
-			HelperFunctions::color(RED);
-			while(!InputStream.eof())
-			{
-				InputStream.getline(buffer,200);
-				cout << buffer << endl;
-			}
-		}
-
-		else
-			cout << "Could not open file";
-		InputStream.close();
-	}
-}
-
-
-
-
-void Game::Play()
-{
-	cout <<"\n\n\n\n";
-	char s[300] = "\t\t\t#####################################################\n"
-				   "\t\t\t#                                                   #\n"
-		           "\t\t\t#                  Text Based Game                  #\n"
-		           "\t\t\t#                                                   #\n"
-		           "\t\t\t#####################################################";
-
-    HelperFunctions::color(RED);
-	cout << s;
-	cout <<"\n\n\n";
-	displayStory();
-
-	char functionName[30];
-	bool exit_Castle = false;
-	playerPtr->Look();
-
-	do
-	{
-		HelperFunctions::color(YELLOW);
-		cout << "\n\nWhat do you want to do : ";
-
-		char fullCommand[30];
-		cin.getline(fullCommand,30);
-
-		//Converting all the characters of fullCommand to UpperCase
-		HelperFunctions::charactersCase(fullCommand);
-
-
-		char command[15];
-
-		int i = 0;
-
-		for (i = 0; fullCommand[i] != '\0' && fullCommand[i] != ' '; ++i)
-		{
-			functionName[i] = fullCommand[i];
-		}
-
-		functionName[i] = '\0';
-
-		//Check for LOOK and EXIT Command's Case
-		if(strlen(functionName) != strlen(fullCommand))
-			strcpy_s(command , 15 , fullCommand + i + 1);
-
-
-		if ( strcmp(functionName,	 "MOVE") == 0)
-		{
-			if (playerPtr-> Move(command , exit_Castle))
-				playerPtr->Look();
-
-			if (exit_Castle == true)
-			{
-					gameCheck();
-			}
-		}
-
-
-		else if ( strcmp(functionName,	 "PICK") == 0)
-			playerPtr->Pick(command);
-
-		else if ( strcmp(functionName,	 "DROP") == 0)
-			playerPtr->Drop(command);
-
-		else if ( strcmp(functionName,	 "LOOK") == 0)
-			playerPtr->Look();
-
-
-		else if ( strcmp(functionName,	 "ATTACK") == 0)
-		{
-			if(playerPtr->Attack(command))
-			{
-				castlePtr->HiddenRoomsUnlocker(command , monstersPtr);
-			}
-			else if(playerPtr->getCurrentState() == false)
-				PlayerDead();
-		}
-
-		else if(strcmp (functionName , "EXIT") == 0)
-		{
-			playerPtr->Exit();
-		}
-		else
-		{
-			HelperFunctions::color(RED);
-			cout << "\nINVALID COMMAND . Please enter a Valid Command .";
```

Comment:
Fix punctuation. Change to `cout << "\nINVALID COMMAND. Please enter a Valid Command.";`